num-to-gpio

```diff
@@ -2,8 +2,8 @@
 # Copyright 2016 Joel Stanley <joel@jms.id.au>
 # Apache 2.0
 
-def num_to_gpio(num):
-    num = int(num) % 320
+def num_to_gpio(base, num):
+    num = int(num) - int(base)
```

Comment:
Before this worked with the base-added or base omitted. Can we still support that?